RestCommClient/Classes/RCDevice.h

```diff
@@ -126,7 +126,10 @@ extern NSString* const RCDeviceCapabilityClientNameKey;
  *    <b>registrar</b>: Restcomm instance to use, like <i>'cloud.restcomm.com'</i>. Leave empty for registrar-less mode <br>
  *    <b>signaling-secure</b>: Should signaling traffic be encrypted? For traffic to be encrypted the 'signaling-certificate-dir' below should be provided as well <br>
  *    <b>signaling-certificate-dir</b>: Directory where files agent.pem and cafile.pem are installed inside the App bundle needed when TLS is to be used for signaling. agent.pem contains the private key and certificate to be used by the signaling facilities. cafile.pem is a file containing all trusted certs
-*     <b>stun-turn-servers</b> NSArray of NSDictionary data of the STUN and TURN servers; This key is mandatory if ICEConfigType is 'kCustom'; Example with 2 servers:
+ *    <b>ice-config-type<b>: Should be integer value of the following  kXirsysV2 = 0, kXirsysV3 =1 , kCustom=2
```

Comment:
Can we add some more doc here, explaining what the kCustom stands for? Let me know if you need help on that. Also I think there is a typo: 1 should be for V3, right? Not V2

Reply:
Yes.

Reply:
Did you add it the additional doc @ognjenns that I asked, so that it's clearer what each does? Let me assist in the writing as the underlying webrtc functionality might be a bit tricky. I 'd say use something like the following:

\<b\>ice-config-type\</b\>: Should be integer value of the following  kXirsysV2 = 0, kXirsysV3 = 1 , kCustom = 2. kXirsysV2 and kXirsysV3 is about utilizing an ICE configuration URL using Xirsys V2 or V3 respectively. **kCustom is about discovering the ICE urls (i.e. STUN and TURN) in the App level and providing to the SDK directly without utilizing an ICE configuration URL. This is handy if you host your own ICE servers, or want more control**